Add helm chart (#227)
openfun/ralph: src/helm/README.md

# Ralph Helm Chart

This is the helm chart used to deploy ralph application.

> All default values are in `values.yaml`

Reviewer: ```suggestion
All default values are in `values.yaml`.

```

Author: Fixed

> ## Review manisfest
> To generate and review your manifest, under `./src/helm`

Reviewer: ```suggestion
## Review manifest

To generate and review your manifest, under `./src/helm` run the following command:

```

```

> helm template .

Reviewer: We usually add the prompt so that users are aware that these are shell commands:

```suggestion
$ helm template .
```

```

## Deploy chart
### Requirements
* Helm
* Needed Kubernetes context selected

### Environments

> Please note that with Helm, you can extend the values files. There is no need to copy/paste all the default values and you can replace a value by setting it in your env file.

Reviewer: ```suggestion

Please note that with Helm, you can extend the values files. There is no need to copy/paste all the default values and you can replace a value by setting it in your env file.
```

> You can add an environments values files under the root of the chart, eg. `dev-values.yaml` and set only the needed customization.

Reviewer: ```suggestion
You can add an environment values file under the root of the chart, _e.g._ `dev-values.yaml` and set only needed customizations.
```

> This chart use the file `vaul.yaml` to set the mandatory secrets for the application

Reviewer: ```suggestion
This chart uses the `vault.yaml` file to set the mandatory secrets for the application.
```

### How to

Under `./src/helm`

```

> helm upgrade --install RELEASE_NAME ralph/. --values ralph/dev-values.yaml

Reviewer: ```suggestion
$ helm upgrade --install RELEASE_NAME ralph/. --values ralph/dev-values.yaml
```

```

> Tips:

Reviewer: ```suggestion
Tips:

```

> * use `--values` to pass an env values files to extend and/or replace the default values

Reviewer: ```suggestion
* use `--values` to pass an env values file to extend and/or replace the default values
```
